Update scrollable-page.html

mikekavouras opened the pull request:
*Issue #, if available:*

*Description of changes:*


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

mikekavouras committed:
Update scrollable-page.html

mikekavouras reviewed (commented):
```suggestion
 Cool, dude
```

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -5,8 +5,8 @@
   <style type="text/css" media="screen">
-    .ace_editor {
-        position: relative !important;
+    .ace_editors {
```
Why did this change?
```suggestion
    .ace_editor {
```

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -5,8 +5,8 @@
-    .ace_editor {
-        position: relative !important;
+    .ace_editors {
+        position: relatively !important;
```
`relatively` isn't a valid css property smh
```suggestion
        position: relative !important;
```

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -136,6 +136,7 @@
 }
 
 function scroll(speed) {
+  // so cool!
```
This can be removed
```suggestion
```

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -152,7 +153,9 @@
 });
+const gutStuff = document.getElementByStuff('guts')
 </script>
+ 
```
✂️

```suggestion
```

mikekavouras committed:
Revert class name change

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -6,7 +6,7 @@
   <style type="text/css" media="screen">
     .ace_editor {
-        position: relative !important;
+        position: relatively !important;
```
`relatively` isn't a valid css property smh

```suggestion
        position: relative !important;
```

mikekavouras committed:
Use a valid css value

mikekavouras committed:
Add some emoji love

mikekavouras committed:
Remove comment

mikekavouras committed:
Remove empty line

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -152,7 +154,9 @@
 });
-</script>
+const gutStuff = document.getElementByStuff('guts')
+ 
```
```suggestion
 
```

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -135,7 +136,8 @@
 
-function scroll(speed) {
+
+  // so cool!
```
```suggestion
```

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -152,7 +154,9 @@
 });
-</script>
+const gutStuff = document.getElementByStuff('guts')
+
```
```suggestion
```

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -152,7 +154,9 @@
 window.scroll = scroll;
 });
-</script>
+const gutStuff = document.getElementByStuff('guts')
```
```suggestion
```

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -31,7 +31,7 @@
 
     .scrollmargin {
-        height: 500px;
+        height: 5000px;
```
```suggestion
        height: 50000px;
```

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -31,7 +31,7 @@
 
     .scrollmargin {
-        height: 500px;
+        height: 50000px;
```
```suggestion
        height: 1000px;
```

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -56,14 +56,17 @@
 <pre id="editor">function foo(items) {
     var i;
     for (i = 0; i &lt; items.length; i++) {
-        alert("Ace Rocks " + items[i]);
```
Look at this!

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -135,7 +138,8 @@
     scroll()
 }
 
-function scroll(speed) {
```
This is a comment

```suggestion

```

mikekavouras replied:
```suggestion
lol
```

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -152,7 +156,9 @@
 window.add = add;
 window.scroll = scroll;
 });
-</script>
```
```javascript
var things = "stuff"
```

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -56,14 +56,17 @@
 <div class="scrollmargin">
-    <span onclick="scroll()" class="large-button">
+    <span onclick="scroll()" class="large-button" onload="return false;">
+      🤖
```
```suggestion
womp
```

mikekavouras replied:
```suggestion
      ❤️
```

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -56,14 +56,17 @@
-    <span onclick="scroll()" class="large-button">
+    <span onclick="scroll()" class="large-button" onload="return false;">
+      🤖
+      🤖
```
comment

mikekavouras replied:
```suggestion
      🦄
```

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -56,14 +56,17 @@
 <div class="scrollmargin">
-    <span onclick="scroll()" class="large-button">
+    <span onclick="scroll()" class="large-button" onload="return false;">
+      ❤️
```
```suggestion
      🦄
```

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -56,14 +56,17 @@
-    <span onclick="scroll()" class="large-button">
+    <span onclick="scroll()" class="large-button" onload="return false;">
+      ❤️
+      🤖
```
```suggestion
      🙉
```

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -56,14 +56,17 @@
 <div class="scrollmargin">
-    <span onclick="scroll()" class="large-button">
+    <span onclick="scroll()" class="large-button" onload="return false;">
+      🦄
```
```suggestion
      🖨
      ❤️
      🛠
```

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -56,14 +56,17 @@
-    <span onclick="scroll()" class="large-button">
+    <span onclick="scroll()" class="large-button" onload="return false;">
+      🦄
+      🙉
```
```suggestion
      👟
```

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -56,14 +56,19 @@
 <div class="scrollmargin">
-    <span onclick="scroll()" class="large-button">
+    <span onclick="scroll()" class="large-button" onload="return false;">
+      🖨
```
```suggestion
      👇🏻
      🗑
```

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -56,14 +56,19 @@
-    <span onclick="scroll()" class="large-button">
+    <span onclick="scroll()" class="large-button" onload="return false;">
+      🖨
+      ❤️
```
```suggestion
      🦃
```

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras committed:
Update demo/scrollable-page.html

mikeghtester commented on demo/scrollable-page.html:
```diff
@@ -56,14 +56,20 @@
-    <span onclick="scroll()" class="large-button">
+    <span onclick="scroll()" class="large-button" onload="return false;">
+      👇🏻
+      🗑
```
This should be an elephant 

```suggestion
      🐘
```

mikeghtester committed:
Love the elephant

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -56,14 +56,20 @@
     var i;
     for (i = 0; i &lt; items.length; i++) {
-        alert("Ace Rocks " + items[i]);
+        alert(`Ace Rocks ${items[i]}`);
```
```suggestion
        alert(`Ace Rocks ${itemssdf[i]}`);
```

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -31,7 +31,7 @@
 
     .scrollmargin {
-        height: 500px;
+        height: 1000px;
```
```suggestion
        height: 100000px;
```

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras committed:
sdf

mikekavouras committed:
Snip snip

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -31,7 +31,7 @@
 
     .scrollmargin {
-        height: 500px;
+        height: 100000px;
```
```suggestion
        height: sdf;
```

mikekavouras committed:
Delete that line!

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras committed:
Delete this line

mikekavouras committed:
Make it a baseball

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -31,7 +31,7 @@
     }
 
     .scrollmargin {
-        height: 500px;
```
```suggestion
  height: 1000px;
```

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -31,7 +31,7 @@
 
     .scrollmargin {
-        height: 500px;
+        height: sdf;
```
```suggestion
        height: sdsdff;
```

mikekavouras replied:
`sdf` isn't a valid value. Try using a number...
```suggestion
        height: 100px;
```

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -56,14 +56,20 @@
 <div class="scrollmargin">
-    <span onclick="scroll()" class="large-button">
+    <span onclick="scroll()" class="large-button" onload="return false;">
+      👇🏻
```
```suggestion
      👨🏻
```

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -56,14 +56,20 @@
     var i;
     for (i = 0; i &lt; items.length; i++) {
-        alert("Ace Rocks " + items[i]);
+        alert(`Ace Rocks ${itemssdf[i]}`);
```
```suggestion
        alert(`Ace Rocks ${items[i]}`);
```

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -56,14 +56,20 @@
+      👨🏻
+      🐘
+      ⚾
+      🛠
```
```suggestion
      🤖
```

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -56,14 +56,20 @@
 <div class="scrollmargin">
-    <span onclick="scroll()" class="large-button">
+    <span onclick="scroll()" class="large-button" onload="return false;">
+      👨🏻
```
```suggestion
      👨🏻
      cool
```

mikekavouras replied:
```suggestion
```

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -135,7 +140,8 @@
 }
 
-function scroll(speed) {
+
```
sdfsdf
```suggestion
sdfsdfgdfh
```

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -56,14 +56,19 @@
 <div class="scrollmargin">
-    <span onclick="scroll()" class="large-button">
+    <span onclick="scroll()" class="large-button" onload="return false;">
+      🐘
```
```suggestion
      ❤️
```

mikekavouras committed:
Heart instead of elephant

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -56,14 +56,19 @@
+    <span onclick="scroll()" class="large-button" onload="return false;">
+      🦃
+      ⚾
+      👟
```
```suggestion
      👟
      🦄
      🛠
```

mikekavouras committed:
Add some buddies

mikekavouras committed:
Use 100px instead of sdf

mikekavouras committed:
Create file.php

mikekavouras commented on file.php:
```diff
@@ -0,0 +1,20 @@
+        parent::__construct($name, $wetsuit);
+        $this->bicycle($bicycle);
+    }
+    public function bicycle(string $bicycle = null): string
```
```suggestion
    private function bicycle(string $bicycle = null): string
```

mikekavouras committed:
make private

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -31,7 +31,7 @@
 
     .scrollmargin {
-        height: 500px;
+        height: 100px;
```
```suggestion
        height: 200px;
```

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras commented on file.php:
```diff
@@ -0,0 +1,20 @@
+<?php
+namespace Acme\Cyclist;
+use Acme\Swimmer\Swimmer;
+class Cyclist extends Swimmer
```
```suggestion
class Cyclist extends Swimmers
```

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -31,7 +31,7 @@
 
     .scrollmargin {
-        height: 500px;
+        height: 200px;
```
```suggestion
        height: 1800px;
```

mikekavouras replied:
NO it should be this 
```suggestion
        height: 900px;
```

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -57,6 +57,7 @@
 <body>
 <div class="scrollmargin">
     <span onclick="scroll()" class="large-button" onload="return false;">
+      🤖
```
```suggestion
```

mikekavouras committed:
Update demo/scrollable-page.html

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -56,14 +56,21 @@
 <div class="scrollmargin">
-    <span onclick="scroll()" class="large-button">
+    <span onclick="scroll()" class="large-button" onload="return false;">
+      🦃
```
```suggestion
      🦃sdf
```

mikekavouras committed:
sdf

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -31,7 +31,7 @@
 
     .scrollmargin {
-        height: 500px;
+        height: 900px;
```
```suggestion
        height: 9800px;
```

mikekavouras committed:
This seems reasonable.

We're changing this value bc it is wrong and needs to be right. Doesn't that make sense?

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -31,7 +31,7 @@
 
     .scrollmargin {
-        height: 500px;
+        height: 9800px;
```
```suggestion
        height: 9sdf00px;
```

mikekavouras committed:
cool!

And even cooler!

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -56,14 +56,21 @@
 <body>
 <div class="scrollmargin">
-    <span onclick="scroll()" class="large-button">
+    <span onclick="scroll()" class="large-button" onload="return false;">
```
```suggestion
    <span onclick="scroll()" class="large-button" onload="return false;">
    
    
```

mikekavouras commented on demo/scrollable-page.html:
```diff
@@ -56,14 +56,21 @@
-    <span onclick="scroll()" class="large-button">
+    <span onclick="scroll()" class="large-button" onload="return false;">
+      🦃sdf
+      ⚾
```
```suggestion
      dog
```

mikekavouras committed:
Apply suggestions from code review

Multiple things
